### ISSUE TITLE
Upgrade dependency of gsl to 2.8

### ISSUE THREAD

**Issue by GoBigorGoHome** [otfried/ipe#514]
Version 2.8 of the [GSL](https://www.gnu.org/software/gsl/) library was released on 25 May, 2024. Now the latest gsl 2.8 is available from some Linux distributions while ipe 7.2.30 requires gsl 2.7. So please consider upgrading dependency of gsl to 2.8.

**Comment by otfried** [comment]
Ipe does not specify any specific dependency, this is up to the person compiling Ipe.  Can you explain what you mean by "upgrading dependency of gsl to 2.8"?

**Comment by GoBigorGoHome** [comment]
Thanks for your quick answer. In fact, I just looked into the ipe source code, and realized the problem I had is more like a packaging issue. 

Let me tell the full story. Today, when I run ipe from command line, there was an error
```
ipe: error while loading shared libraries: libgsl.so.27: cannot open shared object file: No such file or directory
```
I found that the gsl package on my machine had been upgraded to version 2.8, but earlier when Ipe 7.2.30 was installed (built from source), there was gsl 2.7.

I guess the gsl version number 2.7 was "hardcoded" when ipe was built and installed. So Ipe was looking for libgsl.so.27 at start.
